packages/haste-task-express/test/index.spec.js

```diff
@@ -0,0 +1,52 @@
+const http = require('http');
+const { run } = require('haste-test-utils');
+
+const callbackPath = require.resolve('./fixtures/callback');
+const responseText = 'hello world';
+
+const { command: express, kill } = run(require.resolve('../src'));
+
+const request = url => new Promise((resolve, reject) => {
+  const req = http.get(url, (res) => {
+    let data = '';
+
+    res.on('data', (chunk) => {
+      data += chunk;
+    });
+
+    res.on('end', () => resolve({ data, res }));
+  });
+
+  req.on('error', reject);
+});
+
+describe('haste-express', () => {
+  afterEach(kill);
+
+  it('should run an http server', () => {
```

Comment:
I would probably say something about the fact that it accepts a callback path, because that's what we test here.

Maybe, `it('should run an http server and accepts a callback that can manipulate the express app')`

Reply:
I think a callback should always be used, similar to the config being passed on webpack task. There's nothing to do with it without a callback.

I removed it being optional, what do you think?

Reply:
I would probably throw a custom error for not implementing the callback, but I agree it is not functional without it.